### PR TITLE
nixos/mullvad-vpn: add cross-platform availability announcement to re…

### PR DESCRIPTION
**Pull request by arcuru**
…lease notes

###### Description of changes

See #191082 for context.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/doc/manual/release-notes/rl-2211.section.md
Comment (cole-h) [inline nixos/doc/manual/release-notes/rl-2211.section.md:268]: Thanks for handling this!

However, I think the more important thing to call out is that the default package changed -- this means that users who relied on the GUI will need to add the `mullvad-vpn` package to their system (or change the package back) in order to access the GUI.

Maybe something like: "The default package for `services.mullvad-vpn.package` was changed to `pkgs.mullvad`, allowing cross-platform usage of Mullvad. `pkgs.mullvad` only contains the Mullvad CLI tool, so users who rely on the Mullvad GUI will want to change it back to `pkgs.mullvad-vpn`, or add `pkgs.mullvadvpn` to their environment."

Comment (patricksjackson) [reply nixos/doc/manual/release-notes/rl-2211.section.md]: Oops, good call out. I lost some context here since it's been a while since I made the change, but yes that's what we need to warn people about.

I think your suggestion works fine and fits with the rest of the release notes as well. I've updated to use that text.

**Review by cole-h — APPROVED** [review]
Thanks for handling this! I'll merge once ofborg is happy (if I don't get to it after that happens, any other committer can feel free to preempt me).

**Comment by patricksjackson** [comment]
I realized that this change does not make sense without #198116

**Comment by cole-h** [comment]
Thanks for the reminder and that other PR.

**Pull request merged**
